recipes/heyoka.py/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export AR_CMAKE_SETTING=
+    export RANLIB_CMAKE_SETTING=
+else
+    # Workaround for making the LTO machinery work on Linux.
```

Comment:
No LTO on Linux?

Reply:
Do you know which are the correct settings for these variables under OSX? The build system is trying to enable LTO on OSX as well, but failing to do so:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=277482&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1162

Reply:
Ahh now that I am reading the output of the azure builds more carefully, is ``${CMAKE_ARGS}`` supposed to take care of seeing AR, RANLIB, etc. as well? In such case, perhaps I can remove the hack of setting them manually in the Linux builds?

Reply:
No, they use just the generic AR and RANLIB, so you should keep this.

Is there a more detailed error message about OSX?

Reply:
I don't have access to an OSX machine, I would not know how to produce a more detailed error message.

I have tried setting explicitly AR/RANLIB in the same fashion as done in linux, let's see if anything changes...

Reply:
@isuruf the latest commit seems to have done it, but it does not seem like a very solid solution as the AR/RANLIB versions are hard-coded in the script...

Reply:
> I don't have access to an OSX machine, I would not know how to produce a more detailed error message.

You can cross compile if you'd like to.